homeassistant/components/deluge/coordinator.py

```diff
@@ -34,7 +37,7 @@ def __init__(
         self.api = api
         self.config_entry = entry
 
-    async def _async_update_data(self) -> dict[Platform, dict[str, int | str]]:
+    async def _async_update_data(self) -> dict[Platform, dict[str, Any]]:
```

Comment:
`int | str` as value type is likely false. The existing code suggest that at least one type should be a list of sorts.
https://github.com/home-assistant/core/blob/d12857c68cbeb883521b7d7ea28cbb4d2b31f9ae/homeassistant/components/deluge/switch.py#L48-L51

The upstream library isn't typed yet, so I think it makes sense to use `Any` for now.